shell/platform/android/io/flutter/embedding/android/FlutterActivityAndFragmentDelegate.java

```diff
@@ -235,11 +234,16 @@ void onAttach(@NonNull Context context) {
   /**
    * Invoke this method from {@code Activity#onCreate(Bundle)} to create the content {@code View},
    * or from {@code Fragment#onCreateView(LayoutInflater, ViewGroup, Bundle)}.
-   *
-   * <p>{@code inflater} and {@code container} may be null when invoked from an {@code Activity}.
-   *
-   * <p>This method creates a new {@link FlutterView}, adds a {@link FlutterUiDisplayListener} to
-   * it, and then returns it.
+   * <p>
```

Comment:
nit: I expect the formatter is going to complain about this, `./ci/format.sh | patch -p0` should work to fix it. If you install google-java-format in your IDE of choice you should be able to get the changes applied within IntelliJ too. https://github.com/google/google-java-format#intellij-android-studio-and-other-jetbrains-ides

Reply:
Thanks. Formatter installed and run.